content/docs/learning-guide/tutorials/ai/kythera-city-level-overview.md

```diff
@@ -0,0 +1,219 @@
+3. Find an entity in the dropdown menu to see which behavior tree is in use.
+4. Click the `Live` button to the right of the dropdown menu to view the behavior tree in real time
+
+## 2D pathfinding
+
+![A 2D Kythera AI agent](/images/learning-guide/tutorials/ai/kythera-city-robot.png)
+
+2D Navigation is demonstrated by the `Robot.prefab`. The robots use the behavior `2DRandomWalk`, which chooses a random point in a specified range, then finds and follows a path to that point.
+
+When duplicating or moving entities, be sure to select the top (shaded tab) of the prefab in the Entity Outliner, to prevent detached objects in the level.
+
+The `KytRobot` entity in the prefab has the following components, which are required for 2D navigation:
+
+*   `Kythera`
+*   `PhysX Character Controller`, set to be consistent with `navmesh.xml`
+*   `Agent`, set to use the profile `RandomWalk`
+*   `SimpleMovementController`
+The `RandomWalk` profile can be found in `KytheraAIDemo/Scripts/Profiles.xml`:
+
+```xml
+<RandomWalk id="140" inheritanceParent="100" type="bb">
+  <DefaultBehavior type="string">2DRandomWalk</DefaultBehavior>
+</RandomWalk>
+```
+The `DefaultBehavior` name must match the behavior name set in the inspector.
+
+## 3D pathfinding
+
+![A 3D Kythera AI agent](/images/learning-guide/tutorials/ai/kythera-city-spaceship.png)
+
+3D Pathfinding is demonstrated by KytSpacship entities within the level.
+
+The `KytSpaceship` entities use the behavior tree `3DGoToRandomEntityTag`, which searches for a random entity with a Kythera AI tag of `Kyt_Point_of_interest`, then finds and follows a path to that entity.
+
+You can find the `Kythera3D_Point_of_Interest_Tagged` entities in the outliner, which contains a `Kythera` component with an added tag of `Kyt_Point_of_interest`.
+
+!["Point of interest" entity in the Entity Outliner](/images/learning-guide/tutorials/ai/kythera-city-point-of-interest.png)
+
+The octree must be built and present in the level for 3D navigation behavior tree nodes to work.
+
+The `KytSpaceship` entity has the following components, which are required for 3D navigation:
+
+*   `Kythera`
+*   `PhysX Collider`, set to the convex shape of the ship
+*   `Agent`, set to use the profile `GoToRandomEntityProfile`
+*   `FlightMovementController`, which can be configured to change the speed and acceleration of the ships
+*   `PhysX Rigid Body` (required by the movement controller)
+
+The profile `GoToRandomEntityProfile` can be found in `KytheraAIDemo/Scripts/Profiles.xml`.
+```xml
+<GoToRandomEntityProfile id="210" inheritanceParent="200" type="bb">
+  <DefaultBehavior type="string">3DGoToRandomEntityTag</DefaultBehavior>
+</GoToRandomEntityProfile>
+```
+The `DefaultBehavior` name must match the behavior name set in the inspector.
+
+## 3D splines and orientation
+
+![A 3D Kythera AI agent following a navigation spline](/images/learning-guide/tutorials/ai/kythera-city-spaceship.png)
+
+3D splines are demonstrated using a few entities.
```

Comment:
```suggestion
The level demonstrates 3D splines using a few entities.
```
Passive voice fix; confirm that "the level" is accurate as the subject of the sentence.